### ISSUE TITLE
CSP in chrome app with polymer.dart

### ISSUE THREAD

**Issue by Kleak** [dart-lang/polymer-dart#684]
Should be cool if we can use polymer.dart inside a chrome app, actually there are csp issue

**Comment by k4rr1m** [comment]
Use polybuild

**Comment by Kleak** [comment]
yes i could but a transformer will be better, like that all the build process is under pub build ;)

**Comment by jlambright** [comment]
I see documentation and StackOverflow posts referencing the "csp" polymer transformer option but, as of Dart SDK v1.15 & 1.16.0-dev.4.0, pub build throws an invalid transformer option error.

So is csp available for the polymer transformer is not?
